src/link.js

```diff
@@ -4,6 +4,9 @@ import { PUSH, REPLACE } from './action-types';
 
 const LEFT_MOUSE_BUTTON = 0;
 
+const normalizeHref = location =>
+  `${location.basename || ''}${location.pathname}`;
```

Comment:
I'm 95.2345% sure `history` does this under the hood when you call `history.createLocation()`